app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.example.leagueapp.MainActivity">
+    tools:context="com.example.leagueapp.UI.MainActivity">
```

Comment:
BTW we should change our package name to not have "example" on there and be more like a company name like "hongpark"